### PR TITLE
feat: add 'Data Source' to budget search & set budget summary table link 

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
please see order by commits

### CODE REVIEW

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersPopper.vue [hunk 1]
```diff
@@ -37,7 +38,12 @@ const storeState = reactive({
 });
 const state = reactive({
     loading: true,
-    enabledFilters: computed<MenuItem[]>(() => costAnalysisPageStore.defaultGroupByItems.filter((d) => costAnalysisPageState.enabledFiltersProperties?.includes(d.name))),
+    enabledFilters: computed<MenuItem[]>(() => {
+        if (costAnalysisPageStore.selectedQueryId === DYNAMIC_COST_QUERY_SET_PARAMS) {
+            return costAnalysisPageStore.defaultGroupByItems;
+        }
+        return costAnalysisPageStore.defaultGroupByItems.filter((d) => costAnalysisPageState.enabledFiltersProperties?.includes(d.name));
+    }),
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersPopper.vue:46]: description: for here

스크린샷 2023-09-14 오후 1 00 26

**Review by piggggggggy — APPROVED** [review]
LGTM!@ 👍👍👍

File: apps/web/src/services/cost-explorer/budget/budget-detail/modules/budget-summary/BudgetSummaryTable.vue [hunk 2]
```diff
@@ -84,10 +95,10 @@ const getBudgetUsageDataWithRatioAndLink = (accumulatedBudgetData, budgetTimeUni
                 costQuerySetId: DYNAMIC_COST_QUERY_SET_PARAMS,
             },
             query: {
-                granularity: primitiveToQueryString(GRANULARITY.MONTHLY),
+                granularity: primitiveToQueryString(GRANULARITY.DAILY),
                 group_by: arrayToQueryString([GROUP_BY.PRODUCT]),
                 period: objectToQueryString(period),
-                filters: objectToQueryString({ ...costTypeFilters, ...targetFilters }),
+                filters: objectToQueryString(getConvertedConsoleFilters({ ...costTypeFilters, ...targetFilters })),
```
Comment (sulmoJ) [inline apps/web/src/services/cost-explorer/budget/budget-detail/modules/budget-summary/BudgetSummaryTable.vue:101]: praise: ahhh...!

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request merged**